tests/datalab/test_factory.py

```diff
@@ -13,7 +13,14 @@ def registry():
 def test_list_possible_issue_types(registry):
     issue_types = Datalab.list_possible_issue_types()
     assert isinstance(issue_types, list)
-    possible_issues = ["label", "outlier", "near_duplicate", "non_iid", "class_imbalance"]
+    possible_issues = [
```

Comment:
you might wanna have this as a `pytest fixture`.

Reply:
something like this:
```
@pytest.fixture
def possible_issues():
    return ["label", "outlier", "near_duplicate", "non_iid", "class_imbalance"]
```

Reply:
Hi, I think this is not related to this PR, maybe you can create a new PR to improve the pytest?

Reply:
I see, I thought this was a new unit test. 

I am not really sure what these codes are doing, so I would prefer not to contribute without having a good understanding of the code, to be honest. 

you might wanna add a `#TODO` if you think this would take long time.